internal/admin/dashboard/static/css/dashboard.css

```
@@ -723,6 +723,289 @@ body {
+    gap: 14px;
+}
+
+.provider-status-reason {
+    font-size: 13px;
+    color: var(--text-muted);
+}
+
+.provider-status-error {
+    font-size: 12px;
+    color: var(--danger);
+    word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)